### ISSUE TITLE
Range Error when last line contains spaces

### ISSUE THREAD

**Issue by AlexHolly** [standard/standard#1206]
maybe related to https://github.com/standard/standard/issues/1142

**What version of standard?**
npm install standard@10.0.3 --save-dev

**What operating system, Node.js, and npm version?**
Windows 7
node 10.11.0
npm 6.4.1

**What did you expect to happen?**
`> .\node_modules\.bin\standard`
standard: Use JavaScript Standard Style (https://standardjs.com)
  C:\project\src\app.js:1:5: 'test' is defined but never used.

**What actually happened?**
`> .\node_modules\.bin\standard`
Second line contains tab/spaces>0
```
var test
  
```

```
RangeError: Line number out of range (line 3 requested, but only 2 lines present
).
    at SourceCode.getIndexFromLoc (C:\Users\...\AppData\Roaming\npm\node_m
odules\standard\node_modules\eslint\lib\util\source-code.js:389:19)
    at Object.fix (C:\Users\...\AppData\Roaming\npm\node_modules\standard\
node_modules\eslint\lib\rules\no-multiple-empty-lines.js:116:52)
    at RuleContext.report (C:\Users\...\AppData\Roaming\npm\node_modules\s
tandard\node_modules\eslint\lib\rule-context.js:127:34)
    at Program:exit.allLines.reduce.concat.reduce (C:\Users\...\AppData\Ro
aming\npm\node_modules\standard\node_modules\eslint\lib\rules\no-multiple-empty-
lines.js:108:37)
    at Array.reduce (<anonymous>)
    at EventEmitter.Program:exit (C:\Users\...\AppData\Roaming\npm\node_mo
dules\standard\node_modules\eslint\lib\rules\no-multiple-empty-lines.js:93:22)
    at EventEmitter.emit (events.js:187:15)
    at NodeEventGenerator.applySelector (C:\Users\...\AppData\Roaming\npm\
node_modules\standard\node_modules\eslint\lib\util\node-event-generator.js:265:2
6)
...
```

**Comment by LinusU** [comment]
Is there a bug in ESLint about this? If not, could you report one?

**Comment by AlexHolly** [comment]
looks fixed in version 12.0.1
